### DISCUSSION TITLE
Getting error while loading xmlns:graph="clr-namespace:LiveChartsCore.SkiaSharpView.WPF;assembly=LiveChartsCore.SkiaSharpView.WPF"

### DISCUSSION THREAD

**Discussion by Rkjena10** [Live-Charts/LiveCharts2#1493] [Q&A]
I am getting error while loading Livechart2(wpf) from C# winform application like below.

Exception Source: SkiaSharp.Views.WPF
Exception Type: System.TypeInitializationException
Exception Message: The type initializer for 'SkiaSharp.SKImageInfo' threw an exception.
Exception Target Site: OnRender

Inner exception --->
Exception Source: SkiaSharp
Exception Type: System.DllNotFoundException
Exception Message: Unable to load library 'libSkiaSharp'.
Exception Target Site: LoadLocalLibrary

Please help..

**Comment by albertofojo** [comment]
Same error here using .NET8 maui application
